app/src/main/java/com/epfl/beatlink/ui/library/CreateNewPlaylist.kt

```diff
@@ -165,6 +189,9 @@ fun CreateNewPlaylistScreen(
                           playlistTracks = emptyList(),
                           nbTracks = 0)
                   playlistViewModel.addPlaylist(newPlaylist)
+                  if (imageUri != null) {
```

Comment:
I think here should be `imageUri != Uri.EMPTY` as the default value is set to `Uri.EMPTY`